src/components/BottomSheet/BottomSheetMenu.js

```diff
@@ -53,15 +53,15 @@ const styles = StyleSheet.create({
     borderBottomWidth: 1,
     paddingVertical: dimen.normalizeDimen(14),
     paddingHorizontal: dimen.normalizeDimen(20),
-    borderColor: COLORS.greySubtile2
+    borderColor: COLORS.gray200
   },
   rbsheetContent2: {
     flexDirection: 'row',
     alignItems: 'center',
     borderBottomWidth: 1,
     paddingVertical: dimen.normalizeDimen(14),
     paddingHorizontal: dimen.normalizeDimen(20),
-    borderColor: COLORS.greySubtile2
+    borderColor: COLORS.gray200
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor has been updated from `COLORS.greySubtile2` to `COLORS.gray200` for both `rbsheetContent` and `rbsheetContent2`. As with the previous comment, ensure that `COLORS.gray200` is defined and matches the intended color scheme.
```diff
-    borderColor: COLORS.greySubtile2
+    borderColor: COLORS.gray200
```
```diff
-    borderColor: COLORS.greySubtile2
+    borderColor: COLORS.gray200
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->